### PR TITLE
refactor: move appbar

### PR DESCRIPTION
**Pull request by tokitou-san**

### CODE REVIEW

**Comment by baseplate-admin** [comment]
Fix precommit

**Comment by tokitou-san** [comment]
> Fix precommit

No you do? I worked hard now I'm tired

**Review by baseplate-admin — APPROVED** [review]
Add some tests for the appbar layout. God knows when things might break. ( joking aside good work )

File: src/libs/screens/home_screen.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from kivy.utils import get_color_from_hex
+from kivymd.uix.screen import MDScreen
+
+# import components
+from ..components.appbar import appbar
```
[the remark this answers is not in the document]
Comment (tokitou-san) [reply src/libs/screens/home_screen.py:5]:
```
   File "/home/sssuneeth/Desktop/android/src/libs/screens/home_screen.py", line 5, in <module>
     from src.libs.components.appbar import appbar
 ModuleNotFoundError: No module named 'src'
```

[the remark this answers is not in the document]
Comment (baseplate-admin) [reply src/libs/screens/home_screen.py:5]: That's why i told you not to move the main.py into src folder. 

it will be `components.appbar`

**Pull request merged**